content/source/docs/enterprise/private/azure-setup-guide.html.md

```diff
@@ -230,28 +225,28 @@ of the installation guide.
 ### Failure Scenarios
 
 The PTFE Reference Architecture is designed to handle different failure
-scenarios that have different probabilities. As the architecture evolves
-the ability to provide better service continuity may improve.
+scenarios that have different probabilities. The ability to provide better
+service continuity will improve as the architecture evolves.
 
 #### Component Failure
 
 ##### Single VM Failure
 
-In the event of the *PTFE-main* instance failing, the Load Balancer
-should be re-configured (manually or automatically) to route all traffic
-to the *PTFE-standby* instance. It is important to note this
-configuration of routing all traffic to a single instance and not in an
-active-active configuration is by design due to a serialisation
-requirement in the core components of PTFE.
-
-With external services (PostgreSQL Database, Object Storage, Vault) in
-use, there is still some application configuration data present on the
-PTFE server such as installation type, database connection settings,
-hostname. This data rarely changes. If the application configuration has
-not changed since installation, both PTFE-main and PTFE-standby will be
-using the same configuration and no action is required. If the
-configuration on PTFE-main changes you should create a snapshot via the
-UI or CLI and recover this to PTFE-standby so both instances use the
+In the event of the active instance failing, the Load Balancer
+should be reconfigured (manually or automatically) to route all traffic
+to the standby instance.
+
+~> Active-active configuration is not supported due to a serialisation requirement in the core components of PTFE; therefore, all traffic from the Load Balancer *MUST* be routed to a single instance.
```

Comment:
We've been beginning these callouts with a bolded label to indicate what kind of note they are. So: 

* `-> **Note:**`
* `~> **Important:**`

Reply:
Thanks, Nick. I have been looking for this kind of style guide. I can update these.